### PR TITLE
reshuffled schema and data for cold electronics

### PR DESCRIPTION
**Pull request by glehmannmiotto**
this branch takes the latest develop and applies the wibmod schema prepared by @roland-sipos , with minor modifications. In particular it makes the schema compatible with the Hermes approach.
This merge should be done together with the one of wibmod for a successful nightly.

### CODE REVIEW

File: test/config/CRP4-session.data.xml [hunk 1]
```diff
@@ -258,16 +254,14 @@
...
-</obj>
 
 <obj class="WIECApplication" id="hermes-CRP4">
  <attr name="application_name" type="string" val="daq_application"/>
  <rel name="runs_on" class="VirtualHost" id="vlocalhost"/>
  <rel name="contains">
   <ref class="DetectorToDaqConnection" id="crp4-connections"/>
  </rel>
- <rel name="wib_module_conf" class="WIBModuleConf" id="dummy"/>
+ <rel name="wib_module_conf" class="WIBModuleConf" id="def-wib-conf"/>
```
Comment (gcrone) [inline test/config/CRP4-session.data.xml:334]: `def-wib-conf` is not defined anywhere.

Comment (glehmannmiotto) [reply test/config/CRP4-session.data.xml:334]: thanks for spotting this. Fixed.

File: schema/appmodel/wiec.schema.xml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  <attribute name="communication_port" type="u32" init-value="1234" is-not-null="yes"/>
+  <relationship name="settings" class-type="WIBSettings" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
+ </class>
+
+ <class name="WIECApplication" description="Application describing the controller of a detector unit using WIBs inside a WIEC.">
+  <superclass name="SmartDaqApplication"/>
+  <superclass name="ResourceSetAND"/>
+  <relationship name="wib_module_conf" class-type="WIBModuleConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
+  <relationship name="hermes_module_conf" class-type="HermesModuleConf" low-cc="one" high-cc="one" is-composite="no" is-exclusive="no" is-dependent="no"/>
+ </class>
```
Comment (gcrone) [inline schema/appmodel/wiec.schema.xml]: This class should define a `generate_modules()` method. Without it,  src/WIECApplication.cpp will not compile.

Comment (glehmannmiotto) [reply schema/appmodel/wiec.schema.xml]: added the algo, compiled locally and didn't see any issue.

**Review by gcrone — APPROVED** [review]
Looks good now.

**Pull request merged**